sherpa/models/basic.py

```diff
@@ -40,14 +41,34 @@
 __all__ = ('Box1D', 'Const1D', 'Cos', 'Delta1D', 'Erf', 'Erfc', 'Exp', 'Exp10',
            'Gauss1D', 'Log', 'Log10', 'LogParabola', 'NormGauss1D', 'Poisson',
            'Polynom1D', 'PowLaw1D', 'Scale1D', 'Sin', 'Sqrt', 'StepHi1D',
-           'StepLo1D', 'Tan', 'Box2D', 'Const2D', 'Delta2D', 'Gauss2D',
-           'SigmaGauss2D',
+           'StepLo1D', 'Tan', 'Voigt',
+           'Box2D', 'Const2D', 'Delta2D', 'Gauss2D', 'SigmaGauss2D',
            'NormGauss2D', 'Polynom2D', 'Scale2D', 'UserModel', 'TableModel',
            'Integrate1D')
 
 DBL_EPSILON = numpy.finfo(numpy.float).eps
 
 
+class Voigt(RegriddableModel1D):
+    """Voigt Profile
+    https://scipython.com/book/chapter-8-scipy/examples/the-voigt-profile/
+    """
```

Comment:
Should this have a reference for Voigt's 1912 paper where the name comes from? http://publikationen.badw.de/de/003395768

Reply:
Changed the reference:
```
(base) [dtn@devel12 sherpa]$ git diff sherpa/models/basic.py
diff --git a/sherpa/models/basic.py b/sherpa/models/basic.py
index fb118f8c..3f4daa96 100644
--- a/sherpa/models/basic.py
+++ b/sherpa/models/basic.py
@@ -51,12 +51,12 @@ DBL_EPSILON = numpy.finfo(numpy.float).eps
 
 class Voigt(RegriddableModel1D):
     """Voigt Profile
-    https://scipython.com/book/chapter-8-scipy/examples/the-voigt-profile/
+    http://publikationen.badw.de/de/003395768
     """
 ```